feeds.yaml

```diff
@@ -0,0 +1,10 @@
+weBTC-rUSDT:
```

Comment:
if this is test file, please put it somewhere in `test/` dir

Reply:
Removed it